.changeset/fuzzy-crabs-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
+---
+
+Added the `.hero-with-details` hero component variant.
```

Comment:
reminder update change messages to remove .hero-with-details refernece and instead reference `hero-details`

Reply:
I think I get what you are saying here, but we're still needing the one reference in css for `hero-with-details` on the rtl logic, right?  I just wnat to make sure I understand the ask here.

Reply:
I'm not seeing the need for `hero-with-details`. Didn't we removed it? That was the idea behind putting `@include tablet { flex-direction: row }` on line 27 of hero.scss

Reply:
Should we change this to `hero-details` and `hero-details-card`?

Reply:
https://github.com/microsoft/atlas-design/pull/616/files#r1569626135